fork.html

```diff
@@ -0,0 +1,170 @@
+            <li><strong>🔄 Finalize project name</strong>. We need to decide whether to stick with "OpenTF" as a name or pick a different name.</li>  
+            <li><strong>🔄 Finalize foundation & processes</strong>. We need to finalize the details of the OpenTF foundation, including the processes we will use for the steering committee, RFCs, accepting contributions, etc.</li>  
+            <li><strong>🔄 Create initial OpenTF Registry</strong>. HashiCorp recently made some (unannounced) <a href="https://github.com/opentffoundation/roadmap/issues/24#issuecomment-1699535216">changes</a> to the terms of the Terraform Registry, saying it may only be used with Terraform. As a stopgap solution to unblock our develpoment and testing, we are launching an initial OpenTF Registry. We'll develop the official OpenTF Registry solution via an official RFC process later.</li>  
+            <li><strong>🔄 Release process</strong>. Put in place a process for creating OpenTF releases.</li>  
+            <li><strong>🔄 Alpha release</strong>. Once the above items are done, we will create the first OpenTF release. This will be an alpha release, meant for testing by the community.</li>  
+          </ul>
+          <h4 class="small-margin">What's coming soon</h4>          
+          <ul class="tight-list">
+            <li><strong>⏳Start accepting community PRs and RFCs</strong>. Once all the details of the foundation, registry, and release process are in place, we will start accepting contributions from the community!</li>  
+            <li><strong>⏳Create an official OpenTF Registry via an RFC process</strong>. Go through an RFC process to create the official OpenTF Registry solution (replacing the stopgap solution).</li>  
+            <li><strong>⏳Stable release</strong>. Create the first stable OpenTF release. This is meant for production usage, as a drop in replacement for Terraform.</li>  
+          </ul>
+          <h4 class="small-margin">Join the OpenTF community!</h4>          
+          <p class="subtle-margin">
+            The response from the community so far has been incredible. In just a few weeks, more than 130 companies 
+            and 680 individuals have pledged support to the <a href="https://opentf.org/">OpenTF manifesto</a>, and the 
+            <a href="https://github.com/opentffoundation/manifesto">OpenTF manifesto repo</a> has gotten more than 
+            33,000 stars! By comparison, the Terraform repo took nearly 10 years to reach 38,000 stars:
+          </p>  
+          <img src="/images/star-history-202395.png" alt="OpenTF has gotten over 33,000 stars in GitHub in just a few weeks">
+          <p class="subtle-margin">
+            This sort of growth is unprecedent, and we're humbled by all of your support. As per the roadmap in the 
+            previous section, we're working hard on getting the fork to the point where we can start doing releases and
+            accepting community contributions. 
```

Comment:
we're already accepting community contributions the moment the repo is public

Reply:
Updated